README.md

```diff
@@ -176,9 +200,9 @@ A device attribute MUST be one of these:
   </tr>
 </table>
 
-For example, a device with an ID of `686f6d6965` with a `base` and a `windsensor` node would send:
+For example, a device with an ID of `686f6d6965` with a `outdoor-probe` and a `windsensor` node would send:
 
-```
+```cpp
```

Comment:
Actually... now that I think about it, cpp provided the best match but java in combination with quotes might be better. I think we should quote strings anyhow. Btw without checking myself: Is all payload converted to string?

----

cpp:

```cpp
homie/686f6d6965/$homie → 2.1.0
homie/686f6d6965/$name → Weather station
homie/686f6d6965/$localip → 192.168.0.10
homie/686f6d6965/$mac → DE:AD:BE:EF:FE:ED
homie/686f6d6965/$stats/uptime → 120
homie/686f6d6965/$stats/interval → 60
homie/686f6d6965/$fw/name → weatherstation-firmware
homie/686f6d6965/$fw/version → 1.0.0
homie/686f6d6965/$nodes → outdoor-probe,windsensor
homie/686f6d6965/$implementation → esp8266
homie/686f6d6965/$online → true
```

java:

```java
homie/686f6d6965/$homie → 2.1.0
homie/686f6d6965/$name → Weather station
homie/686f6d6965/$localip → 192.168.0.10
homie/686f6d6965/$mac → DE:AD:BE:EF:FE:ED
homie/686f6d6965/$stats/uptime → 120
homie/686f6d6965/$stats/interval → 60
homie/686f6d6965/$fw/name → weatherstation-firmware
homie/686f6d6965/$fw/version → 1.0.0
homie/686f6d6965/$nodes → outdoor-probe,windsensor
homie/686f6d6965/$implementation → esp8266
homie/686f6d6965/$online → true
```

java with "quotes":

```java
homie/686f6d6965/$homie → "2.1.0"
homie/686f6d6965/$name → "Weather station"
homie/686f6d6965/$localip → "192.168.0.10"
homie/686f6d6965/$mac → "DE:AD:BE:EF:FE:ED"
homie/686f6d6965/$stats/uptime → 120
homie/686f6d6965/$stats/interval → 60
homie/686f6d6965/$fw/name → "weatherstation-firmware"
homie/686f6d6965/$fw/version → "1.0.0"
homie/686f6d6965/$nodes → "outdoor-probe,windsensor"
homie/686f6d6965/$implementation → "esp8266"
homie/686f6d6965/$online → true
```